File: src/main/java/com/devwithadi/todolist/Modules/Vertical_recyclerSpacer.java
package com.devwithadi.todolist.Modules;

import android.graphics.Rect;
import android.view.View;

import androidx.annotation.NonNull;
import androidx.recyclerview.widget.RecyclerView;


    public class Vertical_recyclerSpacer extends RecyclerView.ItemDecoration {
        final  int Space;

        public Vertical_recyclerSpacer(int space) {
            Space = space;
        }

        @Override
        public void getItemOffsets(@NonNull Rect outRect, @NonNull View view, @NonNull RecyclerView parent, @NonNull RecyclerView.State state) {
            super.getItemOffsets(outRect, view, parent, state);
            outRect.bottom=Space;

        }
    }

File: src/main/java/com/devwithadi/todolist/Modules/Vertical_recyclerSpacer.kt
package com.devwithadi.todolist.Modules

import android.graphics.Rect
import android.view.View
import androidx.recyclerview.widget.RecyclerView
import androidx.recyclerview.widget.RecyclerView.ItemDecoration

class Vertical_recyclerSpacer(val Space: Int) : ItemDecoration() {
    override fun getItemOffsets(outRect: Rect, view: View, parent: RecyclerView, state: RecyclerView.State) {
        super.getItemOffsets(outRect, view, parent, state)
        outRect.bottom = Space
    }
}